src/main/java/org/grizz/service/UserService.java

```diff
@@ -1,21 +1,41 @@
 package org.grizz.service;
 
 import com.google.common.collect.Sets;
+import lombok.extern.slf4j.Slf4j;
+import org.grizz.exception.UserAlreadyExistException;
 import org.grizz.exception.UserBadPasswordException;
 import org.grizz.model.User;
 import org.grizz.model.repos.UserRepository;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DuplicateKeyException;
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.security.core.userdetails.UserDetails;
 import org.springframework.stereotype.Service;
 
 import static org.springframework.security.crypto.bcrypt.BCrypt.*;
 
+@Slf4j
 @Service
 public class UserService {
+
+    private static int MAX_NUMBER_OF_PLANET = 8;
+    @Autowired
+    private PlanetService planetService;
     @Autowired
     private UserRepository userRepository;
 
+    public void createUser(String login, String password) {
+        try {
+            User newUser = userRepository.insert(User.builder().login(login).passwordHash(hashpw(password, gensalt()))
+                    .planetNumber(0).maxPlanetNumber(MAX_NUMBER_OF_PLANET).build());
+            planetService.createPlanet(newUser);
+            log.info("New player has created with login [" + login + "]");
+        } catch (DuplicateKeyException e) {
+            throw new UserAlreadyExistException(e.getMessage());
```

Comment:
Since we have no transactions for now - such exception-dependent code can introduce some hard to find and replicate errors. Better to check preconditions with read-only operations before making writes.
But sure - we need to investigate mongo transactions ASAP